src/test/resources/json/MANIFEST_bounded_context_one.json

```diff
@@ -0,0 +1,3 @@
+[
+    "file:///home/dylan/IdeaProjects/candy-doc-maven-plugin/target/candyDocTmp/bounded_context_one.json"
```

Comment:
We can not use those kind of absolute path. Those tests can not work from another computer that yours.

Path should be relative to a conventional folder.

Reply:
You're right, I didn't pay attention because the Object Mapper automatically changes the relative path to the absolute path.